unstructured_inference/visualize.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
+# Copyright (c) Megvii Inc. All rights reserved.
+# Modifications copyright (C) 2023 by Unstructured
```

Comment:
I'd add a comment along the lines of:

`# Unstructured modified the original source code found at <link here>`